tests/RobotFramework/tests/tedge_agent/http_server/entity_store_api.robot

```diff
@@ -33,6 +35,65 @@ CRUD apis
     ...    curl -o /dev/null --silent --write-out "%\{http_code\}" http://localhost:8000/tedge/entity-store/v1/entities/device/child01//
     Should Be Equal    ${get}    404
 
+Query api
+    # Build the entity tree:
+    # main
+    # |--- child0
+    # |    |--- child00
+    # |    |    |--- child000
+    # |--- child1
+    # |--- child2
+    # |    |--- child20
+    # |    |--- child21
+    # |    |    |--- child210
+    # |    |    |    |-- child2100
+    # |    |--- child22
+    Register entity    device/child0//    child-device    device/main//
+    Register entity    device/child00//    child-device    device/child0//
+    Register entity    device/child000//    child-device    device/child00//
+    Register entity    device/child1//    child-device    device/main//
+    Register entity    device/child2//    child-device    device/main//
+    Register entity    device/child20//    child-device    device/child2//
+    Register entity    device/child21//    child-device    device/child2//
+    Register entity    device/child210//    child-device    device/child21//
+    Register entity    device/child2100//    child-device    device/child210//
+    Register entity    device/child22//    child-device    device/child2//
+
+    ${output}=    Execute Command    curl http://localhost:8000/tedge/entity-store/v1/entities
+    ${entities}=    Convert String To Json    ${output}
```

Comment:
Better the use the Keyword that you created in the ThinEdgeIO.py library.

```suggestion
    ${entities}=    List Entities
```

Reply:
After that, you can remove the `Collections` and `JSONLibrary` imports at the top of the test file.

Reply:
In that case I would recommend just creating an small test case which just calls the endpoint and checks that the response status code is ok...though thinking about it, I don't think explicitly testing the api endpoint is really required..the ThinEdgeIO is already part of the code so it is closely linked to the test anyway.